classes/fields/boolean.php

```diff
@@ -173,21 +175,20 @@ public function data( $name, $value = null, $options = null, $pod = null, $id =
 	 */
 	public function validate( $value, $name = null, $options = null, $fields = null, $pod = null, $id = null, $params = null ) {
 
+		$required = (int) pods_v( 'required', $options, 0 );
+
+		if ( ! $required ) {
+			// Any value can be parsed to boolean.
```

Comment:
Is this the right decision here?

Reply:
It'd say yes. If we run it through `pre_save` it will always return a `0` or `1` so the required option isn't doing anything really except for `checkbox` type and UI.